README.md

```diff
@@ -111,12 +111,24 @@ This is my original blog post on FaaS from Janurary: [Functions as a Service blo
 
 ### TestDrive
 
-A one-line script is provided to help you get started quickly. You can test-drive FaaS on Docker Swarm with a set of sample functions as defined in the provided [docker-compose.yml](https://github.com/alexellis/faas/blob/master/docker-compose.yml) file. Alternatively if you have a Kubernetes cluster you can [start here](https://github.com/alexellis/faas-netes).
+**Docker Playground**
 
-Use your own laptop or the free community-run Docker playground: play-with-docker.com (PWD) with the button below:
+You can quickly start FaaS on Docker Swarm online using the community-run Docker playground: play-with-docker.com (PWD) by clicking the button below:
 
 [![Try in PWD](https://cdn.rawgit.com/play-with-docker/stacks/cff22438/assets/images/button.png)](http://play-with-docker.com?stack=https://raw.githubusercontent.com/alexellis/faas/master/docker-compose.yml&stack_name=func)
 
+**Docker Swarm**
+
+A set of one-line scripts are provided to help you quickly test-drive FaaS on Docker Swarm with a set of sample functions as defined in the provided [docker-compose.yml](https://github.com/alexellis/faas/blob/master/docker-compose.yml) file.
+
+- `deploy_stack.sh` - for OSX/Linux on x86_64
+- `deploy_stack.armhf.sh` - for Linux on ARM (Raspberry Pi for example)
```

Comment:
Instead of 3 bullets please can this be a single line with the two options mentioned.